package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "private": true,
+  "main": "index.js",
+  "dependencies": {
+    "cloudgov-style": "^0.3.4"
```

Comment:
I might suggest `"cloudgov-style": "~0.3.4"` here. It would keep you at patch versions rather then minor versions. Although we don't intend to put breaking changes in minor versions, we also don't have the best test coverage right now. Thoughts @jeremiak ?